### PR TITLE
docs: improve wording in seo docs

### PR DESCRIPTION
**Pull request by shuvo-me**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have added tests (if possible).
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/1.getting-started/5.seo-meta.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ See [@unhead/schema](https://github.com/unjs/unhead/blob/main/packages/schema/sr
 
 ### Reactivity
 
-Reactivity is supported on all properties, as computed, getters and reactive.
+Reactivity is supported on all properties, as computed, getters, and reactive.
```
Comment (DamianGlowala) [inline docs/1.getting-started/5.seo-meta.md:129]:
```suggestion
Reactivity is supported on all properties, such as computed, getters, and reactive.
```

File: docs/1.getting-started/5.seo-meta.md [hunk 2]
```diff
@@ -170,9 +170,9 @@ It's recommended to use getters (`() => value`) over computed (`computed(() => v
 
 ### Title Template
 
-You can use the `titleTemplate` option to provide a dynamic template for customizing the title of your site. for example, by adding the name of your site to the title of every page.
+You can use the `titleTemplate` option to provide a dynamic template for customizing the title of your site, for example, by adding the name of your site to the title of every page.
 
-The `titleTemplate` can either be a string, where `%s` is replaced with the title, or a function.
+The `titleTemplate` can either be a string, where `%s` is replaced with the title or a function.
```
Comment (danielroe) [inline docs/1.getting-started/5.seo-meta.md:175]: I think the comma makes it clearer.

```suggestion
The `titleTemplate` can either be a string, where `%s` is replaced with the title, or a function.
```

File: docs/1.getting-started/5.seo-meta.md [hunk 3]
```diff
@@ -170,9 +170,9 @@ It's recommended to use getters (`() => value`) over computed (`computed(() => v
 
 ### Title Template
 
-You can use the `titleTemplate` option to provide a dynamic template for customizing the title of your site. for example, by adding the name of your site to the title of every page.
+You can use the `titleTemplate` option to provide a dynamic template for customizing the title of your site, for example, by adding the name of your site to the title of every page.
```
Comment (danielroe) [inline docs/1.getting-started/5.seo-meta.md:173]:
```suggestion
You can use the `titleTemplate` option to provide a dynamic template for customizing the title of your site. For example, you could add the name of your site to the title of every page.
```

File: docs/1.getting-started/5.seo-meta.md [hunk 4]
```diff
@@ -126,7 +126,7 @@ See [@unhead/schema](https://github.com/unjs/unhead/blob/main/packages/schema/sr
 
 ### Reactivity
 
-Reactivity is supported on all properties, as computed, getters and reactive.
+Reactivity is supported on all properties, such as computed, getters, and reactive.
```
Comment (danielroe) [inline docs/1.getting-started/5.seo-meta.md:129]:
```suggestion
Reactivity is supported on all properties, by providing a computed value, a getter, or a reactive object.
```

**Pull request merged**